core/src/main/java/org/elasticsearch/common/ParseField.java

```diff
@@ -29,6 +29,12 @@
  * Holds a field that can be found in a request while parsing and its different variants, which may be deprecated.
  */
 public class ParseField {
+    // todo reuse these commonly used ParseField objects instead of duplicating them in other classes
+    public static final ParseField FIELD_FIELD = new ParseField("field");
```

Comment:
I don't like these being in ParseField because ParseField is broader than Aggregations and is used all over the codebase. I would rather this was moved to a constants interface called `CommonFields` as an inner interface to AggregationBuilder (much like we have `InternalAggregation.CommonFields` for the response side).